Sources/TuistGenerator/Mappers/ResourcesProjectMapper.swift

```diff
@@ -306,14 +306,44 @@ public class ResourcesProjectMapper: ProjectMapping { // swiftlint:disable:this
         // MARK: - Swift Bundle Accessor for Frameworks
         private class BundleFinder {}
         extension Foundation.Bundle {
-        /// Since \(target.name) is a \(
-            target
-                .product
-        ), the bundle for classes within this module can be used directly.
+        /// Since \(target.name) is a \(target.product), \
+        the bundle for classes within this module can be used directly.
         static let module = Bundle(for: BundleFinder.self)
         }
         """
     }
+
+    private static func swiftStaticFrameworkBundleAccessorString(for target: Target) -> String {
```

Comment:
Can we share the common parts between this accessor and `swiftFrameworkBundleAccessorString`? I believe only the candidates should be different?

Reply:
yes, we can. I've taken out the common parts to the `iterateOverCandidatesString`  func